test/new-e2e/pkg/utils/clients/ssh.go

```diff
@@ -160,3 +164,132 @@ func copyFile(sftpClient *sftp.Client, src string, dst string) error {
 	return nil
 
 }
+
+// FileExists create a sftp session to and returns true if the file exists and is a regular file
+func FileExists(client *ssh.Client, path string) (bool, error) {
```

Comment:
I did not want to wrap `ssh.Client` at first but I think it's going to ease a lof of things. Not for this PR, we'll do it after the component refactor.